lib/fx/adapters/postgres.rb

```diff
@@ -125,7 +125,11 @@ def update_trigger(name, on:, sql_definition:)
       #
       # @return [void]
       def drop_function(name)
-        execute "DROP FUNCTION #{name}();"
+        fns = functions.select { |function| function.name == name.to_s }
+
+        fns.each do |function|
+          execute "DROP FUNCTION #{name}(#{function.arguments});"
```

Comment:
Great idea, and that gives us "signature" as a reference to this feature instead of waffling between "definition" and "arguments".